app/gateway/hybrid-mode.md

```diff
@@ -0,0 +1,305 @@
+Control Plane. The Control Plane listens on port [`8005` ({{site.base_gateway}})](/gateway/network-ports-firewall/) or [`443` ({{site.konnect_short_name}})](/konnect-network/) for connections and
+tracks any incoming data from its Data Planes.
+
+Once connected, every API or Kong Manager/{{site.konnect_short_name}} UI action on the Control Plane
+triggers an update to the Data Planes in the cluster.
+
+## Benefits
+
+Hybrid mode deployments have the following benefits:
+
+* **Deployment flexibility:** Users can deploy groups of Data Planes in
+different data centers, geographies, or zones without needing a local clustered
+database for each DP group.
+* **Increased reliability:** The availability of the database doesn't affect
+the availability of the Data Planes. Each DP caches the latest configuration it
+received from the Control Plane on local disk storage, so if [CP nodes are down](/gateway/cp-outage/),
+the DP nodes keep functioning.  
+    * While the CP is down, DP nodes constantly try to reestablish communication.
+    * DP nodes can be restarted while the CP is down, and still proxy traffic
+    normally.
+* **Traffic reduction:** Drastically reduces the amount of traffic to and from
+the database, since only CP nodes need a direct connection to the database.
+* **Increased security:** If one of the DP nodes is compromised, an attacker
+won’t be able to affect other nodes in the {{site.base_gateway}} cluster.
+* **Ease of management:** Admins only need to interact with the CP nodes to
+control and monitor the status of the entire {{site.base_gateway}} cluster.
+
+## Platform compatibility
+
+You can run {{site.base_gateway}} in hybrid mode on any platform where
+{{site.base_gateway}} is [supported](/install/), including [{{site.konnect_short_name}}](https://cloud.konghq.com/).
+
+## Hybrid mode with Kubernetes 
+
+You can run {{site.base_gateway}} on Kubernetes in hybrid mode with or without the [{{site.kic_product_name}}](/kic/). This uses Kubernetes as a runtime for your data planes. 
+
+Running {{site.base_gateway}} in hybrid mode is commonly referred to as "Kong _on_ Kubernetes". Running {{site.base_gateway}} with {{ site.kic_product_name }} is commonly referred to as "Kong _for_ Kubernetes", as it provides a Kubernetes native way of configuring Kong entities using {{ site.kic_product_name }}. Configuring Kong on Kubernetes is identical to deploying {{site.base_gateway}} running on a virtual machine or bare metal.
+
+Configuring a hybrid mode deployment with {{ site.kic_product_name }} should only be used in a small set of circumstances. We recommend using hybrid mode *without* {{ site.kic_product_name }}, or [DB-less mode](/gateway/db-less-mode/) with {{ site.kic_product_name }}, unless you've been otherwise advised by a member of the Kong team.
+
+For the full Kubernetes hybrid mode documentation, see
+[hybrid mode](https://github.com/Kong/charts/blob/main/charts/kong/README.md#hybrid-mode)
+in the `kong/charts` repository.
```

Comment:
@mheap I added a k8s section to this hybrid mode page. I pulled some of the content from the [k8s deployment topologies reference page](https://docs.konghq.com/gateway/latest/production/deployment-topologies/kubernetes/) we already have. My thinking is that I expose the kubernetes info on this page and the db less page, and then call it out on the deployment topologies landing page and call that existing kubernetes page migrated. Thoughts?